cirq/ops/parity_gates.py

```diff
@@ -88,9 +89,11 @@ def _circuit_diagram_info_(self, args: 'cirq.CircuitDiagramInfoArgs'
                               ) -> Union[str, 'protocols.CircuitDiagramInfo']:
         if self._global_shift == -0.5:
             # Mølmer–Sørensen gate.
-            symbol = common_gates._rads_func_symbol(
-                'MS', args,
-                self._diagram_exponent(args, ignore_global_phase=False) / 2)
+            exponent = self._diagram_exponent(args,
+                                              ignore_global_phase=False) / 2
+            pi = sympy.pi if protocols.is_parameterized(exponent) else np.pi
+            angle_str = args.format_radians(radians=exponent * pi)
+            symbol = f'MS({angle_str})'
```

Comment:
This is an eigengate, can it use _format_exponent_as_angle()?

Reply:
Done.